### DISCUSSION TITLE
why  auto create custom_volume_detail.disabled, how to enable?

### DISCUSSION THREAD

**Discussion by leejshades** [NetApp/harvest#3051] [Q&A]
"/opt/harvest2-conf/conf/zapiperf/custom_volume_detail.disabled"  is auto created
how to disabled?

**Comment by rahulguptajss** [comment]
@ybizeul

**Comment by ybizeul** [comment]
If it's `.disabled`, it's not enabled, so what is the issue exactly ?

**Reply by leejshades** [reply]
> If it's `.disabled`, it's not enabled, so what is the issue exactly ?

want to enable volume detail.

**Reply by ybizeul** [reply]
Oh ok, you use this checkbox in Preferences : (Also I strongly suggest you upgrade to NAbox 4)
![image](https://github.com/user-attachments/assets/923499c4-8c6f-4df7-a0a4-49076f23968e)
